pandas/tests/groupby/test_groupby.py

```diff
@@ -2146,3 +2146,16 @@ def test_groupby_column_index_name_lost_fill_funcs(func):
     result = getattr(df_grouped, func)().columns
     expected = pd.Index(["a", "b"], name="idx")
     tm.assert_index_equal(result, expected)
+
+
+@pytest.mark.parametrize("func", ["ffill", "bfill"])
```

Comment:
can you create a test_missing.py (and move other ffill functions to there)

Reply:
Done. I moved every test, which uses only fill functions to the new file.